elastic/integrations
# Airflow initial release

- Enhancement


## What does this PR do?

This PR adds Airflow experimental integration release.

## Checklist

-[ ] Change follows the [contributing guidelines](https://github.com/elastic/integrations/blob/main/CONTRIBUTING.md)
- [x] Supported versions of the monitoring target are documented
- [x] Supported operating systems are documented (if applicable)
- [ ] Integration or [System tests](https://github.com/elastic/elastic-package/blob/master/docs/howto/system_testing.md) exist
- [x] Fields follow [ECS](https://github.com/elastic/ecs) and [naming conventions](https://www.elastic.co/guide/en/beats/devguide/master/event-conventions.html)
- [x] At least a manual test with ES / Kibana / Agent has been performed.
- [x] Required Kibana version set to:


## How to test this PR locally

Add Airflow integration in Kibana, configure airflow host and port, run the agent and check for metrics data in Kibana.



- Closes #https://github.com/elastic/integrations/issues/3748


## Screenshots

<img width="1636" alt="Screenshot 2022-11-13 at 11 34 36 PM" src="https://user-images.githubusercontent.com/102972658/201537132-2ba8f7b1-18b8-4f42-aba1-a91d813976f2.png">
<img width="1631" alt="Screenshot 2022-11-13 at 11 34 06 PM" src="https://user-images.githubusercontent.com/102972658/201537209-8110d4e5-2e5b-4533-8a91-1cd66bd77c53.png">
